kahuna/public/js/search/results.html

```diff
@@ -61,6 +61,17 @@
                     <gr-icon-label gr-icon="clear">Clear selection</gr-icon-label>
                 </a>
 
+                <a id="share-with-url" class="results-toolbar-item results-toolbar-item--right"
+                   ng-class="{'batch-archive__button--disabled': ctrl.selectionCount >= 45}"
+                   ng-click="ctrl.selectionCount < 45 && ctrl.shareSelection()"
+                   ng-if="ctrl.selectionCount > 0"
+                   aria-label="{{ctrl.selectionCount >= 45 ? 'You can't share all these images at once, please reduce to 44 or less' : 'Share Selected Images'}}"
+                   gr-tooltip="{{ctrl.selectionCount >= 45 ? 'You can't share all these images at once, please reduce to 44 or less' : 'Share Selected Images'}}"
```

Comment:
```suggestion
                   gr-tooltip="{{ctrl.selectionCount >= 45 ? 'You can’t share all these images at once, please reduce to 44 or less' : 'Share selected images'}}"
```

Reply:
Just a real apostrophe and capitalisation fix for sentence case (as other tooltips).